course4/homework-string/string.cc

```diff
@@ -6,4 +6,76 @@ string::string()
 {
 }
 
-}
\ No newline at end of file
+string::string(const char * c)
+{
+    int i = 0;
+    while(c[i] != '\0') // I guess we are not supposed to use strlen
+    {
+        chars.push_back(c[i]);
+        ++i;
+    }
+}
+
+string::string(const string& source)
+{
+    chars = source.chars;
+}
+
+string::string(string&& source)
+{
+    chars = std::move(source.chars);
```

Comment:
Is there a point implementing these constructors? The default implementations would do exactly the same.